test/integration/unuthenticated_user_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class UnuthenticatedUserTest < ActionDispatch::IntegrationTest
+  test 'that an unuthenticated user cannot view registerd user and admin data' do
+
+    visit '/orders'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.